metricflow-semantics/metricflow_semantics/experimental/semantic_graph/trie_resolver/group_by_metric_resolver.py

```diff
@@ -345,42 +397,24 @@ def generate(self) -> Mapping[str, MutableDunderNameTrie]:
         """
         current_graph = self._current_graph
         path_finder = self._path_finder
+        simple_metric_nodes = self._simple_metric_nodes
 
-        base_metric_label = SimpleMetricLabel.get_instance()
-        base_metric_nodes = current_graph.nodes_with_labels(base_metric_label)
-
-        if len(base_metric_nodes) == 0:
+        if len(simple_metric_nodes) == 0:
             raise RuntimeError(
                 LazyFormat(
-                    "Did not find any base metric nodes. This indicates an error in graph construction.",
-                    base_metric_nodes=base_metric_nodes,
-                    label=base_metric_label,
-                )
-            )
-
-        if self._verbose_debug_logs:
-            logger.debug(
-                LazyFormat(
-                    "Found base-metric nodes",
-                    base_metric_nodes=base_metric_nodes,
+                    "Did not find any simple-metric nodes. This indicates an error in graph construction.",
```

Comment:
Would this ever get hit if the user has added semantic models to their manifest but no metrics?